llvm/lib/CodeGen/MachineFloatingPointPredicateUtils.cpp

```diff
@@ -0,0 +1,48 @@
+//===- MachineFloatingPointPredicateUtils.cpp -----------------------------===//
```

Comment:
This should be moved into the LLVMGlobalISel library, it's not general codegen

Reply:
at it